src/goto-analyzer/show_callsites.h

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************\
+
+Module: Show symbols that participate in shadowing
+
+Author: Norbert Manthey, nmanthey@amazon.de
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_ANALYZER_SHOW_CALLSITES_H
+#define CPROVER_GOTO_ANALYZER_SHOW_CALLSITES_H
+
+#include "goto-programs/goto_model.h"
```

Comment:
A forward declaration of `class goto_modelt;` would suffice here.

Reply:
will fix.